src/main/java/com/google/sps/servlets/JoinSessionServlet.java

```diff
@@ -22,34 +22,40 @@
 import com.google.sps.data.AttendeeInterface;
 import com.google.sps.data.Attendee;
 
-/** Servlet that returns a list of attendees in a session.  */
-@WebServlet("/join")
+/** 
+ * Servlet that stores users data in datastore and redirected them to the 
+ * session webpage.  
+ */
+@WebServlet("/join-session")
 public class JoinSessionServlet extends HttpServlet {
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response)
+  public void doPost(HttpServletRequest request, HttpServletResponse response)
     throws IOException {
       DatastoreClientInterface datastoreClient = new DatastoreClient();
-      String sessionId = request.getParameter("sessionId");
+      String screenName = request.getParameter("name");
+      String sessionId = request.getParameter("session-id");
```

Comment:
Create a private constant for name and session-id:

SCREEN_NAME_PARAM_KEY = "name"
SESSION_ID_PARAM_KEY = "session-id"